extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -456,30 +474,53 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 }
             }
 
+            function _isDisabledRequest(xhr?: XMLHttpRequestInstrumented, request?: Request | string, init?: RequestInit) {
+                let isDisabled = false;
+                let theUrl:string = (!CoreUtils.isString(request) ? (request as Request).url || "" : request as string).toLowerCase();
+                let idx = theUrl.indexOf("?");
+                if (idx !== -1) {
+                    // Strip off any Query string
+                    theUrl = theUrl.substring(0, idx);
+                }
+
+                // check that this instance is not not used by ajax call performed inside client side monitoring to send data to collector
+                if (!_isNullOrUndefined(xhr)) {
+                    isDisabled = xhr[DisabledPropertyName] === true;
+                } else if (!_isNullOrUndefined(request)) { // fetch
+                    // Look for DisabledPropertyName in either Request or RequestInit
+                    isDisabled = (typeof request === 'object' ? request[DisabledPropertyName] === true : false) ||
+                            (init ? init[DisabledPropertyName] === true : false);
+                }
+
+                if (isDisabled) {
+                    // Add the disabled url if not present
+                    if (!_disabledUrls[theUrl]) {
```

Comment:
Was a little concerned by the key being a full URL (eg. https://xxxxx.xxxx/xxxx etc), however, Chrome, IE (even in IE7 mode) and firefox all correctly handle the url as the key (with dots, slashes etc)

Reply:
Did you manually verified or used some resource for it?

Reply:
Manually verified